index.js

```diff
@@ -38,15 +38,22 @@ async function terraform(args) {
 
 (async () => {
   const terraformDirectory = core.getInput('terraform_directory');
-  let terraformDoApply = core.getInput('terraform_do_apply');
+  let terraformDoApply = core.getInput('terraform_do_apply') === 'true';
+  let terraformDoDestroy = core.getInput('terraform_do_destroy') === 'true';
```

Comment:
Looking at this now, is there any reason for these to not be constants? 🤔

Reply:
`terraformDoApply` can be changed by earlier steps, did the same to destroy for consistency, tbh doesn't really matter to me

Reply:
Ah, right. 👍🏻